recipes/ice-cream/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "ice-cream" %}
+{% set version = "1.10" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/xinehc/ICEcream/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b346db9acfc7a9d339681053c1aa5cc5d1464fb4a9866c137d92e27a55738d1b
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('ice-cream', max_pin='x') }}
+
+requirements:
+  run:
+    - python >=3.6
+    - perl >=5.26
+    - prokka
+    - biopython
+    - aragorn
+    - hmmer
+    - vmatch
+    - blast
+    - prodigal
+    - pandas
+    - r-base >=4.2
+    - r-reshape2 
+    - dna_features_viewer
+
+test:
+  commands:
+    - plotting_script.py --help
+    - ICEcream.sh --help
```

Comment:
If possible, please add a "real" test with some toy dataset. This should catch some of the other possible errors that these tests won't.